README.md

```diff
@@ -292,6 +292,7 @@ Additional lists you might find useful:
 - :strawberry: [CakeDecimal plugin](https://github.com/dereuromark/cakephp-decimal) - A value object approach on handling decimals.
 - :strawberry: [Duplicatable plugin](https://github.com/riesenia/cakephp-duplicatable) - Behavior for duplicating entities including related data.
 - [Fetchable plugin](https://github.com/riesenia/cakephp-fetchable) - Behavior for fetching entities from cache / memory.
+- :strawberry: [Lampager/Lampager plugin](https://github.com/lampager/lampager-cakephp) - Rapid pagination without using OFFSET.
```

Comment:
Plugin name seems to be Lampager/Cake though.

Reply:
Yikes, I missed that detail. Shouldn't review PRs on mobile.

@chitoku-k It's not a good idea using `Cake` in the namespace of your classes, even if it's not top level namespace.

Reply:
I guess the name is okish with the prefix in place^^ just a bit confusing.

Reply:
Thanks for reviewing and please check for #351.

@ADmad Although the convention of this namespace is somehow confusing, the design of `Lampager/Cake` has derived from lampager-cakephp being one of the implementation for Lampager project. There are `Lampager` for a core namespace, `Lampager/Laravel` for a  Laravel implementation, etc, to organize the entire project.